python/src/api_object.py

```diff
@@ -14,7 +14,7 @@ def _get_response(self, path_maybe_with_bookmark):
         """
         response = self.api_object.get_response(path_maybe_with_bookmark)
         unpacked = self.api_object.unpack(response)
-        self.data = unpacked.get('data')
+        self.items = unpacked.get('items') or unpacked.get('data') # items in v5, data in v3
```

Comment:
Perhaps `PagedIterator` should be initialized with this field name? Its creator would have access to the `api_config` which could either carry this name as a configuration attribute or derive it from its version.

Reply:
Done. (There was a test that lazily assumed that either 'items' or 'data' would work, but I fixed the test.)